grades/tasks.py

```diff
@@ -82,6 +82,28 @@ def generate_course_certificates_for_fa_students():
         )
 
 
+@app.task
+def create_combined_final_grades():
+    """
+    Creates any missing CombinedFinalGrades
+    """
+    courses = Course.objects.filter(
+        program__live=True,
+        program__financial_aid_availability=True
+    )
+    for course in courses:
+        if course.has_frozen_runs() and course.has_exam:
+            exam_grades = ProctoredExamGrade.objects.filter(
+                course=course,
+                passed=True,
+                exam_run__date_grades_available__lte=now_in_utc()
+            )
+            users_with_grade = list(CombinedFinalGrade.objects.filter(course=course).values_list('user', flat=True))
```

Comment:
Since this is only used for the user id lookup this should probably be a `set` instead of a `list`